### DISCUSSION TITLE
Portfolio website created with Quasar

### DISCUSSION THREAD

**Discussion by jonafrank13** [quasarframework/quasar#9368] [Show and Tell]
Hi Folks,

Just want to thank the community for helping me build my own profile website, Quasar is simple brilliant, been promoting it everywhere I go :) 

https://jonafrank13.github.io/#/

Any feedback, comments, critique welcome

**Comment by hawkeye64** [comment]
Nice site! (◍•ᴗ•◍)❤ it!

**Reply by JokerMartini** [reply]
great stuff

**Comment by johnrhanzel** [comment]
Nice portfolio

**Comment by JokerMartini** [comment]
Great site, my only feedback is 'do not completely remove the native cursor from being drawn'. I think that is more of a hinderance than a benefit. However i love when users have cool whispy things that follow the cursor around like you have.
